pkg/server/plugin/nodeattestor/x509pop/x509pop_test.go

```diff
@@ -272,18 +314,37 @@ func (s *Suite) TestGetPluginInfo() {
 	require.Equal(resp, &plugin.GetPluginInfoResponse{})
 }
 
-func (s *Suite) configure(extraConfig string) {
+func (s *Suite) configure(config string) {
 	resp, err := s.p.Configure(context.Background(), &plugin.ConfigureRequest{
-		Configuration: fmt.Sprintf(`
-ca_bundle_path = %q
-%s
-`, s.rootCertPath, extraConfig),
-		GlobalConfig: &plugin.ConfigureRequest_GlobalConfig{TrustDomain: "example.org"},
+		Configuration: config,
+		GlobalConfig:  &plugin.ConfigureRequest_GlobalConfig{TrustDomain: "example.org"},
 	})
 	s.Require().NoError(err)
 	s.Require().Equal(resp, &plugin.ConfigureResponse{})
 }
 
+func (s *Suite) createConfiguration(bundlePathType, extraConfig string) string {
+	switch bundlePathType {
+	case "ca_bundle_path":
+		return fmt.Sprintf(`
+ca_bundle_path = %q 
+%s
+`, s.rootCertPath, extraConfig)
+
+	case "ca_bundle_paths":
+		bundlesPath := fmt.Sprintf("[%q,%q]", s.alternativeBundlePath, s.rootCertPath)
+		return fmt.Sprintf(`
+ca_bundle_paths = %s 
+%s
+`, bundlesPath, extraConfig)
+
+	default:
+		s.FailNow("Usupported bundle path, %q", bundlePathType)
```

Comment:
This won't do what you think it will. Assuming the path type "BAD", it would emit something like:
```
                Error:          Unsupported bundle path, %q
                Test:           Suite/SomeTest
                Messages:       BAD
```

You want something like:
```suggestion
		s.FailNow("Unsupported bundle path type", "type=%q", bundlePathType)
}
```